proc.c

```diff
@@ -2365,6 +2367,9 @@ rb_method_entry_arity(const rb_method_entry_t *me)
  *       def four(a, b); end
  *       def five(a, b, *c);    end
  *       def six(a, b, *c, &d); end
+ *       def seven(a:, b:); end
```

Comment:
What I like in the Proc#arity examples is that they use x, y, z for keyword args and a, b, c for „normal“ ones (at least in the first batch of examples); that makes it easier to read. What do you think?

Reply:
@stomar thanks! That's a good point, I've updated the pr and also updated Proc#arity docs, so it'd be more consistent.